checkov/main.py

```diff
@@ -80,6 +81,11 @@ def run(banner=checkov_banner, argv=sys.argv[1:]):
     external_checks_dir = get_external_checks_dir(args)
     url = None
 
+    all_checks = BaseCheckRegistry.get_all_registered_checks()
+    ckv_to_bc_id_mapping = bc_integration.get_ckv_to_bc_id_mapping()
```

Comment:
Maybe we should cache that mapping on a file with 1 day TTL. It will save a lot of REST API call time. WDYT?

Reply:
Yeah, I think the biggest improvement would be for VSCode. Otherwise I would say one API call per run is not that significant.

The call is already happening; we're not adding an additional call here. So I would prefer to merge this and then handle this change as a new PR (perhaps attached to my in-progress integrations refactor?). Does that work?